pom.xml

```diff
@@ -156,6 +156,12 @@
       <version>0.25.0</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-inline</artifactId>
+      <version>2.24.0</version>
```

Comment:
Let's use newer version of the library i.e. `4.11.0` if it's compatible with java 8, otherwise we can use latest from 3.x series.